draft-barnes-acme.md

```diff
@@ -375,10 +375,10 @@ to clients, and requiring any signed request from the client to carry
 such a nonce.
 
 An ACME server MUST include an Replay-Nonce header field in each
-successful response to a POST it provides to a client, with contents as specified
-below.  It MAY also provide one in an error response.  The value
-provided in this header MUST be unique to this response, with high
-probability.
+successful response it provides to a client, with contents as specified
+below.  It MAY also provide one in an error response.  Providing nonces
+in response to all requests enables a client to obtain a fresh nonce
+by sending a HEAD requests to any ACME resource.
```

Comment:
I find the wording slightly confusing. Does this allow to retrieve the nonce through `GET` as well? Also, it seems to contradict `MAY` in `MAY also provide one in an error response`.